composer/trainer/checkpoint_hparams.py

```diff
@@ -75,13 +75,20 @@ class CheckpointSaverHparams(hp.Hparams):
     folder: str = hp.optional(doc="Folder in which to save checkpoint files. Relative to the run directory, if set."
                               "Defaults to `checkpoints`.",
                               default="checkpoints")
+    compression: str = hp.optional(doc="Compression algorithm to run on checkpoints. Must be one of `None`, `gzip`, `bzip2`, or `lzma`.",
```

Comment:
Let's use the empty string to represent no compression -- it's a bit weird to have strings of `None`.

```suggestion
    compression: str = hp.optional(doc="Compression algorithm to run on checkpoints. Must be blank or one of `gzip`, `bzip2`, or `lzma`.",
```